context/app/static/js/components/detailPage/entityHeader/EntityHeaderContent/EntityHeaderContent.tsx

```diff
@@ -237,7 +237,7 @@ function EntityHeaderContent({ view, setView }: { view: SummaryViewsType; setVie
       <RightDiv>
         {vizIsFullscreen ? (
           <>
-            {vizNotebookId && <VisualizationWorkspaceButton uuid={vizNotebookId} />}
+            {vizNotebookId && <VisualizationWorkspaceButton />}
```

Comment:
Sorry, I attached the previous comment to the wrong snippet. Isn't the `uuid` needed here?